loops.js

```diff
@@ -11,7 +11,11 @@
 // Dit heb je nog niet geleerd, maar bekijk hiervoor dit MDN-artikel eens: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat
 // ==========================================
 
-
+console.log("Opdracht 1.");
+for (let i = 0; i < 5; i++) {
+    const starCount = "*";
```

Comment:
Hier heb je een variabele aangemaakt genaamd starCount. Dat is wel netjes, maar niet per se nodig hier.
Geen idee of het conventie is, maar ik vind het wel mooier dan zonder variabele ^^.